docs/modules/plugins/pages/plugin-mobitru.adoc

```diff
@@ -43,6 +43,42 @@ selenium.grid.capabilities.platformVersion=12
 selenium.grid.capabilities.deviceName=iPhone 13
 ----
 
+For more fine-grained configuration of taking device additionally use
+Mobitru device search capabilities specified below.
+
+NOTE: The properties marked with *bold* are mandatory.
+
+[cols="6,2,5", options="header"]
+
+|===
+
+|Property Name
+|Acceptable values
+|Description
+
+|`*selenium.grid.capabilities.mobitru-device-search\:type*`
+|`phone` or `tablet`
+|The type of the device.
+
+|`selenium.grid.capabilities.mobitru-device-search\:manufacturer`
+|`ZTE` or `samsung` or `IPHONE` etc.
+|The device manufacture name (case-insensitive).
+
+|`selenium.grid.capabilities.mobitru-device-search\:model`
+|`SM-G930F` or `sm-s901b` etc.
+|The model of the device (case-insensitive).
+|===
```

Comment:
I believe this part should be in the properties section of the plugin documentation

Reply:
previously was asked to move it here